float64 xy_min
float64 xy_max
planner/ObstacleMsg[] obstacles
